### PR TITLE
feat(ui): Simplify blue colors

### PR DESCRIPTION
**Pull request by billyvg**
Continuation of the simplification of theme colors for dark mode. This changes all uses of blue4/500 to blue300. 100/200 were not in use previously.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/contextData.jsx [hunk 1]
```diff
@@ -271,9 +271,9 @@ const ToggleIcon = styled('a')`
   margin-left: 1px;
   border-radius: 2px;
 
-  background: ${p => (p.isOpen ? p.theme.gray500 : p.theme.blue400)};
+  background: ${p => (p.isOpen ? p.theme.gray500 : p.theme.blue300)};
   &:hover {
-    background: ${p => (p.isOpen ? p.theme.gray600 : p.theme.blue500)};
+    background: ${p => (p.isOpen ? p.theme.gray600 : p.theme.blue300)};
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/contextData.jsx:276]: @doralchan What would you suggest here for this hoverstate?

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/getCrumbDetails.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ function getCrumbDetails(type: BreadcrumbType) {
 
     case BreadcrumbType.INFO:
       return {
-        color: 'blue400',
+        color: 'blue300',
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/getCrumbDetails.tsx:45]: Hmm maybe to differentiate from `QUERY`

```suggestion
        color: 'blue200',
```

**Comment by markstory** [comment]
Do the new alert boxes still have enough contrast to be meet accessibility targets?

**Comment by billyvg** [comment]
@markstory Chrome thinks so 
image

**Review by doralchan — APPROVED** [review]

**Pull request merged**